src/modules/ordersTable/pure/OrderStatusBox/index.tsx

```diff
@@ -60,6 +60,32 @@ const Wrapper = styled.div<{
   }
 `
 
+function getOrderStatusTitle(order: ParsedOrder): string {
```

Comment:
Actually, there was a comment "Cancelled status takes precedence" but it wasn't true, `isCancelling` had a higher priority. But now everything is in its place.
We need it, because we don't reset `isCancelling` flag for TWAP orders.
Issue to fix it: https://github.com/cowprotocol/cowswap/issues/2634

Reply:
That was a bug fix for an edge case.
Can't find out now which one, but anyway. We should test this bit thoroughly to make sure this isn't breaking anything

Reply:
Thank you! 
Added test instructions for @elena-zh